<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>842</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomQPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>490</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(20, 47, 78, 219), stop:1 rgba(85, 98, 112, 226));
	color:rgba(255, 255, 255, 210);
	border-radius:5px;
}
QPushButton#pushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(40, 67, 98, 219), stop:1 rgba(105, 118, 132, 256));
}
QPushButton#pushButton:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(105, 118, 132, 200);
}</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Downloads/caret-up-solid.svg</normaloff>../../../Downloads/caret-up-solid.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QCustomQPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>560</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_2{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(20, 47, 78, 219), stop:1 rgba(85, 98, 112, 226));
	color:rgba(255, 255, 255, 210);
	border-radius:5px;
}
QPushButton#pushButton_2:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(40, 67, 98, 219), stop:1 rgba(105, 118, 132, 256));
}
QPushButton#pushButton_2:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(105, 118, 132, 200);
}</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Downloads/caret-right-solid.svg</normaloff>../../../Downloads/caret-right-solid.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QCustomQPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>630</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_3{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(20, 47, 78, 219), stop:1 rgba(85, 98, 112, 226));
	color:rgba(255, 255, 255, 210);
	border-radius:5px;
}
QPushButton#pushButton_3:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(40, 67, 98, 219), stop:1 rgba(105, 118, 132, 256));
}
QPushButton#pushButton_3:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(105, 118, 132, 200);
}</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Downloads/caret-down-solid.svg</normaloff>../../../Downloads/caret-down-solid.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QCustomQPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>560</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_4{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(20, 47, 78, 219), stop:1 rgba(85, 98, 112, 226));
	color:rgba(255, 255, 255, 210);
	border-radius:5px;
}
QPushButton#pushButton_4:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(40, 67, 98, 219), stop:1 rgba(105, 118, 132, 256));
}
QPushButton#pushButton_4:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(105, 118, 132, 200);
}</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Downloads/caret-left-solid.svg</normaloff>../../../Downloads/caret-left-solid.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>800</y>
      <width>582</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Roboto Light</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(220, 220, 220);
background-color: rgb(98, 98, 162);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>&quot;If you control the code, you control the world.&quot;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="AnalogGaugeWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>180</y>
      <width>431</width>
      <height>331</height>
     </rect>
    </property>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>16</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>-40</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(220, 220, 220);</string>
     </property>
     <property name="text">
      <string>0°</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(2, 207, 196);</string>
    </property>
    <property name="text">
     <string>0°</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openB">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#openB{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(20, 47, 78, 219), stop:1 rgba(85, 98, 112, 226));
	color:rgba(255, 255, 255, 210);
	border-radius:5px;
}
QPushButton#openB:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(40, 67, 98, 219), stop:1 rgba(105, 118, 132, 256));
}
QPushButton#openB:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(105, 118, 132, 200);
}</string>
    </property>
    <property name="text">
     <string>OPEN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeB">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>50</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#closeB{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(20, 47, 78, 219), stop:1 rgba(85, 98, 112, 226));
	color:rgba(255, 255, 255, 210);
	border-radius:5px;
}
QPushButton#closeB:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(40, 67, 98, 219), stop:1 rgba(105, 118, 132, 256));
}
QPushButton#closeB:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(105, 118, 132, 200);
}</string>
    </property>
    <property name="text">
     <string>CLOSE</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnalogGaugeWidget</class>
   <extends>QWidget</extends>
   <header>AnalogGaugeWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QCustomQPushButton</class>
   <extends>QPushButton</extends>
   <header>QCustomQPushButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
